client/components/jetpack/daily-backup-status/status-card/backup-successful.jsx

```diff
@@ -99,6 +105,8 @@ const BackupSuccessful = ( { backup, selectedDate, lastBackupAttemptOnDate } ) =
 				rewindId={ actionableRewindId }
 				isMultiSite={ isMultiSite }
 				hasWarnings={ hasWarnings }
+				availableActions={ availableActions }
+				onClickClone={ onClickClone }
```

Comment:
Need to pass the rewindId to onClickClone( rewindId ) to get it back to the clone-flow handler.